v2/apiserver/schemas/project-role-assignment.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "github.com/brigadecore/brigade/v2/project-role-assignment.schema.json",
+
+	"definitions": {
+
+    "apiVersion": {
+			"type": "string",
+			"description": "The major version of the Brigade API with which this object conforms",
+			"enum": ["brigade.sh/v2"]
+		},
+
+		"kind": {
```

Comment:
Spacing looks off here... this definitions section looks to have 4 space indents as opposed to the section below?  Or maybe just GitHub display weirdness?

Reply:
I will check.

Reply:
Our JSON schemas have consistently used tabs. Some spaces snuck in here instead.

To avoid the religious was over tabs v spaces, I'll state that I have no preference for one over the other and instead usually opt for whatever is considered conventional for any given language or file format. (Go, for instance, says to use tabs.)

I'm not aware their being any established convention for tabs v spaces in JSON, so I'm going to change these spaces to tabs to be consistent with the rest of this file and with all our other JSON files.

Reply:
Fixed

Reply:
Ah ok; Yea, no pref here either.  This sounds good.  I think I have my vs code setting at "intuit from file whether to insert tabs v spaces"